scrape.py

```diff
@@ -0,0 +1,28 @@
+import urllib2
+
+source = "https://data.consumerfinance.gov/api/views/s6ew-h6mp/rows.csv?accessType=DOWNLOAD"
+tmp_name = "data/cfpb_ccdb.csv"
+
+#url = "http://download.thinkbroadband.com/10MB.zip"
+#file_name = url.split("/")[-1]
+
+u = urllib2.urlopen(source)
```

Comment:
why not just use wget?

https://pypi.python.org/pypi/wget

am i missing something?  you're trying to download the source csv file into 'data/cfpb_ccdb.csv'?